app.js

```diff
@@ -0,0 +1,189 @@
+                    let selectedCount = 0;
+                    checkboxes.forEach(cb => {
+                        if (cb.checked) {
+                            const dataIndex = cb.value
+                            const usdArray = []
+                            for (let i = 1; i <= 10; i++) {
+                                usdArray.push(allData[dataIndex].priceUsd)
+                            }
+                            const newData = {
+                                data: usdArray,
+                                label: allData[dataIndex].name,
+                                borderColor: colorScheme[selectedCount],
+                                fill: false
+                            }
+                            addData(tokenChart, newData)
+                            selectedCount++;
+                        }
+                        else {
+                            removeData(tokenChart, selectedCount)
+                        }
+                    })
+                    checkboxes.forEach(cb => {
+                        if (!cb.checked && selectedCount >= maxSelections) {
+                            cb.disabled = true;
+                        } else {
+                            cb.disabled = false;
+                        }
+                    });
+                    if (selectedCount === maxSelections) {
+                        alert(`Beware, unwary traveler: you have just reached the realm of ${maxSelections} choices limit. In order to select another choice you will have to disable one of the choices you have already chosen first and yet continue unscathed.`);
+                        selectedCount--;
+                    }
+                })
+            });
+        };
+
+        document.getElementById("unfollow-all").addEventListener('click', () => {
+            const checkboxesNode = document.querySelectorAll('input[type="checkbox"]');
+            const checkboxes = Array.from(checkboxesNode);
+            checkboxes.forEach(cb => {
+                if (cb.checked) {
+                    cb.checked = false;
+                }
+                cb.disabled = false;
+            });
+            annihilateData(tokenChart)
+        });
+
+        const tokensHTML = generateTokensList(tokens);
+
+        const renderHTML = (html, target) => document.getElementById(target).innerHTML = html;
+
+        const renderTokensHTML = html => renderHTML(html, 'token-box');
+
+        renderTokensHTML(tokensHTML);
+
+        switcherFunction();
+    }
+
+    try {
+        document.getElementById('token-box').innerHTML =
+            `<fieldset></><img src = "https://media.tenor.com/beTSEyCoetcAAAAM/money-money-money-make-it-rain.gif"></img>
+        <legend>Refresh the page and the data would be shown</legend></fieldset>`;
+        tokenImplementation();
+    }
+
+    catch (error) {
+        console.log(error);
+    }
+
+    finally {
+        tokenImplementation();
```

Comment:
not clear. you run the same function in the `try` block and then again in `finally`?